playwright.config.ts

```diff
@@ -0,0 +1,39 @@
+import dotenv from "dotenv";
+
+import { defineConfig, devices } from "@playwright/test";
+
+dotenv.config();
+
+export default defineConfig({
+  testDir: "playwright/tests",
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: "html",
+  // webServer: {
+  //   command: "npm run dev",
+  //   url: "http://localhost:9000/",
+  //   reuseExistingServer: !process.env.CI,
+  // },
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    baseURL: process.env.CYPRESS_baseUrl,
```

Comment:
question: why not `CYPRESS_BASEURL`?

Reply:
TBH it was left for consistency - we use same names in workflows and tests so I did not want to introduce changes in this PR - I will create separate PR to clean it up